packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6063,12 +6063,18 @@ class IndexedSemantics extends SingleChildRenderObjectWidget {
   }) : assert(index != null),
        super(key: key, child: child);
 
+  factory IndexedSemantics.wrap(int index ,Widget child, Key key) {
```

Comment:
nit: extra whitepspace after `index` and missing whitespace after the comma.

Reply:
removed this function